src/hotspot/os/aix/os_aix.cpp

```diff
@@ -273,6 +273,23 @@ julong os::Aix::available_memory() {
   }
 }
 
+jlong os::total_swap_space() {
+  perfstat_memory_total_t memory_info;
+  if (libperfstat::perfstat_memory_total(NULL, &memory_info, sizeof(perfstat_memory_total_t), 1) == -1) {
+    return -1;
+  }
+  return (jlong)(memory_info.pgsp_total * 4L * 1024L);
```

Comment:
please use 4 * K instead of 4L * 1024L. K is declared as size_t in globalDefinitions.hpp.